src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final int MAX_VALUE = 10;
+    private static final int DEFAULT_FIGURE_VALUE = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+
+    public Figure getRandomFigure() {
+        final int randomSize = random.nextInt(MAX_VALUE) + 1;
+        FigureType[] figures = FigureType.values();
+
+        switch (figures[random.nextInt(figures.length)]) {
```

Comment:
Minor thing here: note that the `figures` variable (btw, it should be named `figureTypes`) is not needed elsewhere and the only thing that interests you in this part of code is a random figure type. 
That means you can improve the code readibility by extracting this code into a separate method. 

Think of a appropriate name for it, for example: `getRandomFigureType()`.